app/controllers/documents_controller.rb

```diff
@@ -197,14 +197,19 @@ def course_list
   end
 
   def update
+    if session[:lms_authenticated_user] | @document.organization[:enable_anonymous_actions] | has_role("designer")
```

Comment:
are these binary or? should it be `||`

Reply:
yea it should. good thing you caught that but when i tested, it was working i will fix this tomorrow